tests/cleanup/remove_old_objects_openstack.sh

```diff
@@ -0,0 +1,121 @@
+#!/bin/bash
+# Script removes virtual machines and other artifacts older than HOURS_LIMIT (24 hours by default) from OpenStack
+
+. /usr/share/beakerlib/beakerlib.sh
+
+
+rlJournalStart
+    rlPhaseStartSetup
+        if [ -z "$OS_AUTH_URL" ]; then
+            rlFail "OS_AUTH_URL is empty!"
+        else
+            rlLogInfo "OS_AUTH_URL=$OS_AUTH_URL"
+        fi
+
+        if [ -z "$OS_USERNAME" ]; then
+            rlFail "OS_USERNAME is empty!"
+        else
+            rlLogInfo "OS_USERNAME is configured"
+        fi
+
+        if [ -z "$OS_PASSWORD" ]; then
+            rlFail "OS_PASSWORD is empty!"
+        else
+            rlLogInfo "OS_PASSWORD is configured"
+        fi
```

Comment:
Ansible will need `OS_TENANT_NAME`, which usually matches `OS_USERNAME` see the test script in `tests/cli`. Isn't this failing for you without this env variable defined ?

Reply:
that's strange but if it works ok then